packages/sourcecred/src/ui/components/LedgerViewer/LedgerViewer.js

```diff
@@ -95,12 +102,63 @@ export const LedgerViewer = ({
     }
   );
 
+  const handleChangeStartDate = useCallback((date) => {
+    setStartDateFilter(date);
+  }, []);
+
+  const handleChangeEndDate = useCallback((date) => {
+    setEndDateFilter(date);
+  }, []);
```

Comment:
after thinking more about this, why not just pass the function as is, if it is for future additions to logic that can def be added later but for now, why not just pass `setStartDateFilter` and `setEndDateFilter` as is?

Reply:
done